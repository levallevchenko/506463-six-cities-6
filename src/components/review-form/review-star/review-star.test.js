import React from 'react';
import {render} from '@testing-library/react';
import ReviewStar from './review-star';

it(`ReviewStar should render correctly`, () => {

  const testProps = {
    id: 1,
    onInputChange: jest.fn(),
    rating: `5`,
    title: ``,
    checked: ``,
  };

  const {container} = render(
      <ReviewStar testProps={testProps} />
  );

  expect(container).toMatchSnapshot();
});
