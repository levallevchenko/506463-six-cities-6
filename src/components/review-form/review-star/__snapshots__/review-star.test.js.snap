// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReviewStar should render correctly 1`] = `
<div>
  <input
    class="form__rating-input visually-hidden"
    name="rating"
    type="radio"
    value=""
  />
  <label
    class="reviews__rating-label form__rating-label"
  >
    <svg
      class="form__star-image"
      height="33"
      width="37"
    >
      <use
        xlink:href="#icon-star"
      />
    </svg>
  </label>
</div>
`;
